crates/ddl/src/core/compile/doc/mod.rs

```diff
@@ -0,0 +1,209 @@
+use codespan::FileId;
+use codespan_reporting::diagnostic::Diagnostic;
+use std::borrow::Cow;
+use std::io;
+use std::io::prelude::*;
+
+use crate::core;
+
+pub fn compile_module(
+    writer: &mut impl Write,
+    module: &core::Module,
+    report: &mut dyn FnMut(Diagnostic),
+) -> io::Result<()> {
+    let context = ModuleContext {
+        _file_id: module.file_id,
+    };
+
+    const MINIRESET: &str = include_str!("./minireset.min.css");
+    const STYLE: &str = include_str!("./style.css");
+
+    let pkg_name = env!("CARGO_PKG_NAME");
+    let pkg_version = env!("CARGO_PKG_VERSION");
+
+    write!(
+        writer,
+        "\
```

Comment:
Maybe use a raw string here to allow writing a bit more naturally (no need to escape quotes for example).

Reply:
Good idea. This might be better solved with a better templating language though... so I think I'm fine with just doing escaping for now! 😅

Reply:
Decided to go with the raw string literals after all!